feature-flags.md

```diff
@@ -0,0 +1,42 @@
+## `data_loss_protect`
+
+Accepted in November 2017: [meeting log](https://docs.google.com/document/d/1x2Qr2VgqauFfPgs-uBv7B4GegifAzRmfA_LpebvsETM/edit)
+
+This option is required by most implementations, e.g. LND: [brontide.go:2749](https://github.com/lightningnetwork/lnd/blob/cc0321d1881ed23c9608cf898af2e2a7b347304a/peer/brontide.go#L2749).
+
+## `static_remote_key`
+
+This removes key-rotation from the `to_remote` output. E.g. watchtowers are easier to implement this way.
```

Comment:
I would also add that it makes it easier to recover most of your funds in case you lost your channels backup.